lib/glific/third_party/bigquery/bigquery_schema.ex

```diff
@@ -1293,6 +1293,162 @@ defmodule Glific.BigQuery.Schema do
     ]
   end
 
+  @doc """
+  Schema for message broadcast contacts table
+  """
+  @spec message_broadcast_contacts_schema :: list()
+  def message_broadcast_contacts_schema do
+    [
+      %{
+        description: "Unique ID for the message broadcast contacts",
+        name: "id",
+        type: "INTEGER",
+        mode: "REQUIRED"
+      },
+      %{
+        description: "Reference for the message broadcast",
+        name: "message_broadcast_id",
+        type: "INTEGER",
+        mode: "NULLABLE"
+      },
+      %{
+        description: "Phone number of the contact",
+        name: "phone",
+        type: "STRING",
+        mode: "NULLABLE"
+      },
+      %{
+        description: "Status of Broadcast",
+        name: "status",
+        type: "STRING",
+        mode: "NULLABLE"
+      },
+      %{
+        description: "Time when the message broadcast contact was first created",
+        name: "inserted_at",
+        type: "DATETIME",
+        mode: "REQUIRED"
+      },
+      %{
+        description: "Time when the message broadcast contact was last updated",
+        name: "updated_at",
+        type: "DATETIME",
+        mode: "REQUIRED"
+      },
+      %{
+        description: "Unique UUID for the row (allows us to delete duplicates)",
+        name: "bq_uuid",
+        type: "STRING",
+        mode: "NULLABLE"
+      },
+      %{
```

Comment:
@AkhileshNegi Procesed_at time is also required here. 
Sometime we process it and then it got failed.

Reply:
@pankaj-ag added additional fields